src/main/java/com/sfuentes/ticket/Ticket.java

```diff
@@ -0,0 +1,35 @@
+package com.sfuentes.ticket;
+
+import com.sfuentes.grocery.Client;
+import com.sfuentes.grocery.Grocery;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class Ticket {
+
+  @NonNull
```

Comment:
Why @NonNull in this property?

Reply:
I got wrong, I thought that if I used "final and @NonNull" I could specify the fields in constructor. But I think that I should have used only "final or @NonNull", not both, I'm really not sure if it's the correct way yet.  I took it from this link: https://stackoverflow.com/questions/27547171/how-can-i-specify-a-one-argument-constructor-using-lombok